superset/cli/importexport.py

```diff
@@ -176,20 +177,21 @@ def import_datasources(path: str) -> None:
     from superset.commands.dataset.importers.dispatcher import ImportDatasetsCommand
     from superset.commands.importers.v1.utils import get_contents_from_bundle
 
-    if is_zipfile(path):
-        with ZipFile(path) as bundle:
-            contents = get_contents_from_bundle(bundle)
-    else:
-        with open(path) as file:
-            contents = {path: file.read()}
-    try:
-        ImportDatasetsCommand(contents, overwrite=True).run()
-    except Exception:  # pylint: disable=broad-except
-        logger.exception(
-            "There was an error when importing the dataset(s), please check the "
-            "exception traceback in the log"
-        )
-        sys.exit(1)
+    with override_user(user=security_manager.find_user(username="admin")):
```

Comment:
it seems it would probably make sense to add a `username` option here and leave `admin` as the default. @betodealmeida what to you think?

Reply:
From a consistency point of view, this makes sense to me. Let me know if you'd like me to go ahead and try with `--username` as an optional option.

Reply:
Agreed with Daniel's feedback there

Reply:
Hey @ddxv wondering if you had a chance to look at this last change. So tempted to merge it :)

Reply:
@geido Are you referring to Daniel's suggestion? I'm all for that. Do we need to wait for any confirmation from @betodealmeida? Just to be sure, I didn't see any recent "change" to code though, so let me know if I am missing something here.

Reply:
@geido I went ahead and pushed the change suggested by Daniel for an optional field with `default="admin"`. My only thought on that is that if you want to match `import_dashboards` it would be a required field without a default. This might be better just to have their API be the same for documentation and use, but let me know what you think.

Reply:
@ddxv it would be great if you want to open a PR to match `import_dashboards` unless there is any opposition. For the time being I am approving this PR and moving forward.